Makefile

```diff
@@ -30,20 +30,29 @@ PYTEST=poetry run pytest -v
 # 	`make test TEST_ARGS="-k some_filter"`
 TEST_ARGS=
 
+CLOUD_FILTERS = -m "not run_only_for_edge_endpoint"
+EDGE_FILTERS = -m "not skip_for_edge_endpoint"
+
 test: install  ## Run tests against the prod API (needs GROUNDLIGHT_API_TOKEN)
-	${PYTEST} ${TEST_ARGS} test
+	${PYTEST} ${TEST_ARGS} ${CLOUD_FILTERS} test
+
+test-4edge: install  ## Run tests against the prod API via the edge-endpoint (needs GROUNDLIGHT_API_TOKEN)
+	${PYTEST} ${TEST_ARGS} ${EDGE_FILTERS} test
 
 test-local: install  ## Run tests against a localhost API (needs GROUNDLIGHT_API_TOKEN and a local API server)
-	GROUNDLIGHT_ENDPOINT="http://localhost:8000/" ${PYTEST} ${TEST_ARGS} test
+	GROUNDLIGHT_ENDPOINT="http://localhost:8000/" ${PYTEST} ${TEST_ARGS} ${CLOUD_FILTERS} test
 
 test-integ: install  ## Run tests against the integ API server (needs GROUNDLIGHT_API_TOKEN)
-	GROUNDLIGHT_ENDPOINT="https://api.integ.groundlight.ai/" ${PYTEST} ${TEST_ARGS} test
+	GROUNDLIGHT_ENDPOINT="https://api.integ.groundlight.ai/" ${PYTEST} ${TEST_ARGS} ${CLOUD_FILTERS} test
+
+test-integ-4edge: install  ## Run tests against the integ API server (needs GROUNDLIGHT_API_TOKEN)
```

Comment:
Just as a side note, we currently don't have support for integ on the edge since [NGINX](https://github.com/groundlight/edge-endpoint/blob/9b7f393509e57dd39a24e9c50a1e7982335d7e79/configs/nginx.conf#L22) is set up to proxy every failed request to prod. Nothing is stopping us from changing this though.